modules/declarative/pages/annotations.adoc

```diff
@@ -0,0 +1,119 @@
+include::{mod-loc}attributes.adoc[]
+
+//Category: skupper-annotations
+// Type: assembly
+[id="skupper-annotations"] 
+= Configuring services using annotations
+
+Using Kubernetes annotations allows you to control which services are available on the {service-network}.
+
+
+// Type: procedure
+[id="exposing-services-annotations"] 
+== Exposing simple services on a {service-network} using annotations
+
+This section provides an alternative to the `skupper expose` command, allowing you to annotate existing resources to expose simple services on the {service-network}.
+
+.Prerequisites
+
+* A site with a service you want to expose
+
+.Procedure
+
+. Log into the namespace in your cluster that is configured as a site.
+
+. Create a deployment, some pods, or a service in one of your sites, for example:
++
+----
+$ kubectl create deployment hello-world-backend --image quay.io/skupper/hello-world-backend
+----
++
+This step is not {skupper-name}-specific, that is, this process is unchanged from standard processes for your cluster.
+
+. Annotate the kubernetes resource to create a service that can communicate on the {service-network}, for example:
++
+--
+----
+$ kubectl annotate deployment backend "skupper.io/address=backend" "skupper.io/port=8080" "skupper.io/proxy=tcp"
+----
+
+The required annotations are:
+
+* `skupper.io/address` - the name of the service on the {service-network}. 
+* `skupper.io/port` - the port for the service on the {service-network}. 
+* `skupper.io/proxy` - the protocol you want to use, `tcp`, `http` or `http2`.
```

Comment:
Actually informing just the `skupper.io/proxy` is enough (in many cases).
Maybe it is worth mentioning that, but also keeping the other two so that
users have a better control on what is being exposed.